website/source/docs/internals/discovery-chain.html.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: "docs"
+page_title: "Discovery Chain"
+sidebar_current: "docs-internals-discovery-chain"
+description: |-
+  The service discovery process can be modeled as a "discovery chain" which passes through three distinct stages: routing, splitting, and resolution. Each of these stages is controlled by a set of configuration entries.
+---
+
+# Discovery Chain <sup>beta</sup>
+
+-> We recommended reading the high-level documentation for [L7 Traffic
+Management](/docs/connect/l7-traffic-management.html) first. This page
+discusses lower-level aspects that make up the implementation of that feature
+set.
+
+The service discovery process can be modeled as a "discovery chain" which
+passes through three distinct stages: routing, splitting, and resolution. Each
+of these stages is controlled by a set of [configuration
+entries](/docs/agent/config_entries.html).
+
+-> **NOTE** The discovery chain is only used to discover
```

Comment:
should we add "currently" to this? It's at least somewhat intended to use this for HTTP/DNS eventually... OK with leaving it though given how non-concrete that timeline is...

Reply:
I would added "currently" or try to soften the language. For example: "The discover chain enables users to discover proxy upstream services so that......." I'd also add this to the previous paragraph.